src/main/universal/res/config/simplify_expr.jspec

```diff
@@ -13,6 +13,10 @@ class simplify_expr {
   boolean b1,b2;
   int e1;
 
+  axiom ideq { (b1 == b1) == true }
```

Comment:
Bob remarked at some point that vercors does not support correct pattern matching in duplicate names (I'm not sure whether that's true), we should check that before admitting this axiom.